modules/core/TextureWood.h

```diff
@@ -0,0 +1,43 @@
+//
+//  Created by Mahmoud El Bergui on 21.04.22.
+//
+
+
+#include"Texture.h"
+
+namespace {
```

Comment:
The declaration of the methods in anonymouse namespace is not needed. Just delete this block from the header file.

Reply:
I will delete them